.travis.yml

```diff
@@ -21,53 +23,108 @@ addons:
 
 env:
     global:
+
         # The following versions are the 'default' for tests, unless
-        # overidden underneath. They are defined here in order to save having
+        # overridden underneath. They are defined here in order to save having
         # to repeat them for all configurations.
+        - PYTHON_VERSION=3.6
+        - NUMPY_VERSION=stable
```

Comment:
@nmearl - I would recommend just commenting out those builds for now, and I can open a PR later to enable and fix them. There are some underlying ci-helpers issues that need to be fixed.